packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -8,6 +8,12 @@ import styles from './Layout.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+
+  /**
+  * Object of configuration for the site alert.
+  */
+  siteAlert: PropTypes.string,
```

Comment:
so if this is being passed into layout as an object, how do you forsee something like dex using this? like a redux store that some lower level component triggers to update and adds a siteAlert into the store, which then propogates to this and renders the alert?

We can worry about this later, but layout wraps everything, so changing a prop will cause the whole page to re-render. I think we thought through this and solved it in theme provider with a middle container around the children content that specifies shouldcomponentupdate

Reply:
Site alerts are different than content alerts. A site alert is like an announcement for a scheudled downtime... it will be configured in the CMS and not change dynamically via interacting with app. 

But you do bring up a good concern.. and I would like to talk more about it